project/Deps.sc

```diff
@@ -373,9 +375,9 @@ object Izumi {
       ),
       Artifact(
         name = Projects.fundamentals.bio,
-        libs = allMonadsOptional ++ Seq(
-            scala_java_time in Scope.Test.js
-          ),
+        libs = allMonadsOptional ++ Seq(monix_bio in Scope.Compile.all) ++ Seq(
```

Comment:
Could you make monix `Optional`, like zio?

Reply:
Done.